apps/condo/domains/ticket/tasks/exportTicketsTask.js

```diff
@@ -0,0 +1,182 @@
+        executor: ticket.executor || '',
+        assignee: ticket.assignee || '',
+        organizationComments: renderedOrganizationComments.join(TICKET_COMMENTS_SEPARATOR),
+        residentComments: renderedResidentComments.join(TICKET_COMMENTS_SEPARATOR),
+        source: ticket.source || '',
+        deadline: ticket.deadline ? formatDate(ticket.deadline) : '',
+        reviewValue: ticket.reviewValue ? reviewValueText[ticket.reviewValue] : '',
+        reviewComment: ticket.reviewComment || '',
+        statusReopenedCounter: ticket.statusReopenedCounter || '',
+    }
+}
+
+const buildExportFile = async ({ rows, locale, idOfFirstTicketForAccessRights }) => {
+    const { stream } = await _buildExportFile({
+        templatePath: './domains/ticket/templates/TicketsExportTemplate.xlsx',
+        replaces: {
+            tickets: rows,
+            i18n: {
+                ...getHeadersTranslations(EXPORT_TYPE_TICKETS, locale),
+                sheetName: i18n('excelExport.sheetNames.tickets', { locale }),
+
+                // These statuses are needed to set cell color using conditional formatting by status name.
+                statusNames: ticketStatusesTranslations(locale),
+            },
+        },
+    })
+    return {
+        stream,
+        filename: `tickets_${dayjs().format('DD_MM')}.xlsx`,
+        mimetype: EXCEL_FILE_META.mimetype,
+        encoding: EXCEL_FILE_META.encoding,
+        meta: {
+            listkey: 'Ticket',
+            // Id of first record will be used by `OBSFilesMiddleware` to determine permission to access exported file
+            // NOTE: Permissions check on access to exported file will be replaced to checking access on `ExportTicketsTask`
+            id: idOfFirstTicketForAccessRights,
+        },
+    }
+}
+
+const exportTickets = async (taskId) => {
+    if (!taskId) throw new Error('taskId is undefined')
+    const { keystone: context } = await getSchemaCtx('TicketExportTask')
+
+    const task = await TicketExportTask.getOne(context, { id: taskId })
+
+    const statuses = await TicketStatus.getAll(context, {})
+    const indexedStatuses = Object.fromEntries(statuses.map(status => ([status.type, status.name])))
+
+    let idOfFirstTicketForAccessRights
+
+    await exportRecords({
+        context,
+        loadRecordsBatch: (offset, limit) => {
+            const tickets = loadRecordsBatch({ context, task, offset, limit })
+            if (!idOfFirstTicketForAccessRights) {
+                idOfFirstTicketForAccessRights = get(tickets[0], 'id')
+            }
+            return tickets
+        },
+        convertRecordToFileRow: (ticket) => convertRecordToFileRow({ task, ticket, indexedStatuses }),
+        buildExportFile: (rows) => buildExportFile({ rows, locale: task.locale, idOfFirstTicketForAccessRights }),
+        task,
+        taskServerUtils: TicketExportTask,
+    })
+}
+
+const exportTicketsTask = createTask('exportTickets', async (taskId) => {
```

Comment:
please make sure to run this in low priority in order not to block sending urgent messages like SMS